tests/unit/models/blueprint-test.js

```diff
@@ -490,66 +553,158 @@ help in detail');
+            if (err) {
+              throw 'error';
+            }
+            expect(content).to.match(/I AM TESTY/);
+          });
+        }).not.to.throw();
 
-        });
+      });
     });
 
     it('re-installing identical files', function() {
       return blueprint.install(options)
-        .then(function() {
-          var output = ui.output.trim().split(EOL);
-          ui.output = '';
+      .then(function() {
+        var output = ui.output.trim().split(EOL);
+        ui.output = '';
+
+        expect(output.shift()).to.match(/^installing/);
+        expect(output.shift()).to.match(/create.* .ember-cli/);
+        expect(output.shift()).to.match(/create.* .gitignore/);
+        expect(output.shift()).to.match(/create.* bar/);
+        expect(output.shift()).to.match(/create.* foo.txt/);
+        expect(output.shift()).to.match(/create.* test.txt/);
+        expect(output.length).to.equal(0);
 
-          expect(output.shift()).to.match(/^installing/);
-          expect(output.shift()).to.match(/create.* .ember-cli/);
-          expect(output.shift()).to.match(/create.* .gitignore/);
-          expect(output.shift()).to.match(/create.* bar/);
-          expect(output.shift()).to.match(/create.* foo.txt/);
-          expect(output.shift()).to.match(/create.* test.txt/);
-          expect(output.length).to.equal(0);
+        return blueprint.install(options);
+      })
+      .then(function() {
+        var actualFiles = walkSync(tmpdir).sort();
+        var output = ui.output.trim().split(EOL);
+
+        expect(output.shift()).to.match(/^installing/);
+        expect(output.shift()).to.match(/identical.* .ember-cli/);
+        expect(output.shift()).to.match(/identical.* .gitignore/);
+        expect(output.shift()).to.match(/identical.* bar/);
+        expect(output.shift()).to.match(/identical.* foo.txt/);
+        expect(output.shift()).to.match(/identical.* test.txt/);
+        expect(output.length).to.equal(0);
+
+        expect(actualFiles).to.deep.equal(basicBlueprintFiles);
+      });
+    });
 
-          return blueprint.install(options);
-        })
-        .then(function() {
-          var actualFiles = walkSync(tmpdir).sort();
-          var output = ui.output.trim().split(EOL);
+    it('re-installing conflicting files', function() {
+      return blueprint.install(options)
+      .then(function() {
+        var output = ui.output.trim().split(EOL);
+        ui.output = '';
+
+        expect(output.shift()).to.match(/^installing/);
+        expect(output.shift()).to.match(/create.* .ember-cli/);
+        expect(output.shift()).to.match(/create.* .gitignore/);
+        expect(output.shift()).to.match(/create.* bar/);
+        expect(output.shift()).to.match(/create.* foo.txt/);
+        expect(output.shift()).to.match(/create.* test.txt/);
+        expect(output.length).to.equal(0);
+        var blueprintNew = new Blueprint(basicNewBlueprint);
```

Comment:
maybe worthwhile comment:

There are subtle difference in `fn.__hasSuper` value i noticed. They make sense if you add `this._super.apply(this, arguments)` below line 287 here https://github.com/ember-cli/ember-cli/blob/6ada827ad30bb0368ddf04fb125eb925c0dfbce8/lib/models/blueprint.js#L287

then console.log the following here: https://github.com/ember-cli/ember-cli/blob/6ada827ad30bb0368ddf04fb125eb925c0dfbce8/tests/unit/models/blueprint-test.js#L628

```js
        console.log("basicNewBlueprint %o", basicNewBlueprint);
        console.log("basicNewBlueprint.constructor %o", basicNewBlueprint.constructor);
        console.log("basicNewBlueprint.constructor.prototype %o", basicNewBlueprint.constructor.prototype);

        console.log("blueprintNew %o", blueprintNew);
        console.log("blueprintNew.constructor %o", blueprintNew.constructor);
        console.log("blueprintNew.constructor.prototype %o", blueprintNew.constructor.prototype);

        var theBluePrint2 = new Blueprint(basicBlueprint);
        console.log("theBluePrint2 %o", theBluePrint2);
        console.log("theBluePrint2.constructor %o", theBluePrint2.constructor);
        console.log("theBluePrint2.constructor.prototype %o", theBluePrint2.constructor.prototype);
```

this has to do with `InstrumentedBasicBlueprint` not implementing init, but with this_super in init method on Blueprint, `fn.__hasSuper` becomes true even when init is not implemented on a intermediate object in the chain - thought that was worth a note, but played last night first time, ignore if not relevant (or not fully described, it was late, but fun)